appshell/appshell_extensions_platform.h

```diff
@@ -97,6 +98,21 @@ int32 ShowSaveDialog(ExtensionString title,
                        ExtensionString initialDirectory,
                        ExtensionString proposedNewFilename,
                        ExtensionString& newFilePath);
+#else
+void ShowOpenDialog(bool allowMulitpleSelection,
```

Comment:
Yeah I thought of that. It is just that we need to unnecessarily process arguments and return types for the sake of unification.